Vagrantfile

```diff
@@ -16,7 +16,11 @@ ensure_plugins(vconfig.fetch('vagrant_plugins')) if vconfig.fetch('vagrant_insta
 
 trellis_config = Trellis::Config.new(root_path: ANSIBLE_PATH)
 
-Vagrant.require_version '>= 2.1.0', '<= 2.2.19'
```

Comment:
@cmurtagh you had previously added an `=` to support Ansible 2.2.19, which caused a merge conflict. I resolved it by accepting the upstream changes seen below

Reply:
Changes should be safe, since I don't think we're using Darwin in any of our environments :joy: